src/js/stores/schemes/healthChecksRowScheme.js

```diff
@@ -1,12 +1,15 @@
 var Util = require("../../helpers/Util");
 
 var HealthCheckProtocols = require("../../constants/HealthCheckProtocols");
+var HealthCheckPortTypes = require("../../constants/HealthCheckPortTypes");
 
 const healthChecksRowScheme = {
   protocol: HealthCheckProtocols.HTTP,
   command: null,
   path: null,
+  port: 0,
```

Comment:
Should we really assume a `port` of 0 in our scheme by default? Knowing that `port` and `portIndex` can't live on the same definition at the same time this seems like a pitfall.